stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2889,22 +2945,35 @@ struct _Native${Self}Storage<${TypeParametersDecl}> :
 
   @_versioned
   internal var startIndex: Index {
-    return Index(nativeStorage: self, offset: -1).successor()
+    // This looks weird because of the FIXME in index(after:)
```

Comment:
The comment is wrong. I was going to stick a line note there, but got distracted. Sorry!